stateproof/builder.go

```diff
@@ -242,7 +242,10 @@ func (spw *Worker) initBuilders() {
 		spw.builders[rnd] = buildr
 	}
 
-	spw.ledger.RegisterVotersCommitListener(spw)
+	err = spw.ledger.RegisterVotersCommitListener(spw)
+	if err != nil {
+		spw.log.Errorf("initBuilders: registering voters listener failed: %v", err)
```

Comment:
Format errors using %w to pack them.

Reply:
this is not creating an error but logging. wrapping does not make sense